flask_resty/filtering.py

```diff
@@ -12,7 +14,10 @@
 # -----------------------------------------------------------------------------
 
 # Field.missing is deprecated in favor of Field.load_default in marshmallow 3.13.0
-_USE_LOAD_DEFAULT = marshmallow.__version_info__ >= (3, 13)
+_MARSHMALLOW_VERSION = importlib.metadata.version("marshmallow")
+_USE_LOAD_DEFAULT = packaging.version.parse(
```

Comment:
Using `packaging.version` for proper semantic version check. *Should* be OK I think? This handles https://peps.python.org/pep-0440/ vs full Semver I believe...